deepmd/dpmodel/fitting/general_fitting.py

```diff
@@ -73,7 +73,10 @@ class GeneralFitting(NativeOP, BaseFitting):
             different fitting nets for different atom types.
     exclude_types: List[int]
             Atomic contributions of the excluded atom types are set zero.
-
+    remove_vaccum_contribution: List[bool], optional
```

Comment:
how should one write the `remove_vaccum_contribution` if `mixed_types`?  like [True]?  It seems that True is better.

Reply:
This option is for internal use, called by the energy fitting (which users define `atom_ener`), so only the developers will touch it.

`torch.jit` doesn't like `Union[bool, List[bool]]`.